audformat/core/utils.py

```diff
@@ -1622,9 +1625,32 @@ def job(file: str) -> pd.Timedelta:
                 progress_bar=verbose,
                 task_description='Read duration',
             )
-            ends.values[idx_nat] = durs
```

Comment:
Since the proposed solution looks slighly hacky I wonder if we should simply do instead:

```python
            ends = ends.to_series()
            ends[idx_nat] = durs
```